micro_mg3_0.F90

```diff
@@ -1081,11 +1115,24 @@ subroutine micro_mg_tend ( &
   real(r8) :: irad
   real(r8) :: ifrac
 
-  real(r8) :: nimey  !meyers ice nucleation
-  real(r8) :: niact(mgncol,nlev) ! dummy for modified activation
+  !Variables for accretion seeing autoconverted liquid    
+  real(r8) :: rtmp(mgncol,nlev) ! dummy for rain + autoconversion
+  real(r8) :: ctmp(mgncol,nlev) ! dummy for liq - autoconversion
+  real(r8) :: ntmp(mgncol,nlev) ! dummy for liq - autoconversion number
 
+  ! Variables for height calculation
+  real(r8) :: zhalf(mgncol,nlev)  !midpoint height
```

Comment:
shall we initialize the `zhalf` variable?

Reply:
Done